app/packages/core/src/plugins/SchemaIO/components/TextView.tsx

```diff
@@ -0,0 +1,59 @@
+import { Box, Typography } from "@mui/material";
+import React from "react";
+import { getComponentProps } from "../utils";
+import { NumberSchemaType, ViewPropsType } from "../utils/types";
+
+export default function TextView(props: ViewPropsType<NumberSchemaType>) {
+  const { schema } = props;
+  const { view = {} } = schema;
+  const {
+    color = "primary",
+    fontSize = "1rem",
+    title = "",
+    textTransform = "none",
+    variant = "body1",
+    bold = false,
+    italic = false,
+    align = "inherit",
+    noWrap = false,
+    textDecoration = "none",
```

Comment:
@manivoxel51 Sorry for the late review. We should support snake case for everything that is set from Python side

Reply:
you are right! will patch it shortly 🙇